chinese-beautifulsoup.Rmd

```diff
@@ -0,0 +1,201 @@
+# Chinese Translation--Beautiful Soup Tutorial
+
+## Haifeng Lan
```

Comment:
The ## will turn this line into 3.1 (for the example that your cc will be merged into chapter 3), just double checking do you want you name to be a subheader? It would make more sense to leave your name without the ##.